52/souhailshiv/Pomodor.py

```diff
@@ -0,0 +1,65 @@
+import os
+from time import *
+import beepy
+import keyboard
+from tkinter import *
+from tkinter import ttk
+
+root = Tk()
+
+w = Label(root, text="this is pomodor ", fg="red",
+          bg="black", font=("Helevetica", 20))
+w.pack()
+
+
+def pomodor(min):
+    print("work time")
+    sec = 0
+    while min != 0 or sec != 0:
+        print("minuts: ", min, " ", "seconds: ", sec)
+        sleep(1)
+        if sec == 0:
+            if min != 0:
+                min = min-1
+                sec = 59
+            else:
+                break
+        elif keyboard.is_pressed("s"):
+            exit()
+        else:
+            sec = sec-1
+
+    beepy.beep(sound=5)
+    Break(xy)
+
+
+def Break(minut):
+    print("break time")
+    second = 0
+    while True:
+        print("minuts: ", minut, " ", "seconds: ", second)
+        sleep(1)
+        if second == 0:
+            if minut != 0:
```

Comment:
maybe easier to convert to seconds? then you save some math here

Reply:
maybe

Reply:
maybe

Reply:
I would convert to seconds, makes it much easier